tbxforms/templates/tbx/field.html

```diff
@@ -49,7 +49,7 @@
             {% include "tbx/layout/help_text_and_errors.html" %}
             {% crispy_tbx_field field %}
             {% if max_characters or max_words %}
-                <span id="{{ field.id_for_label }}-info" class="tbxforms-hint tbxforms-character-count__message" aria-live="polite">
+                <p id="{{ field.id_for_label }}-info" class="tbxforms-hint tbxforms-character-count__message" aria-live="polite">
```

Comment:
Is the idea with `tbxforms` that projects can upgrade the version they are working with? If so this could introduce some breaking layouts as paragraphs tend to have default bottom margins applied (not to mention being a block level element when spans are not).

Reply:
`.tbxforms-hint` has it's own [margin defintion](https://github.com/torchbox/tbxforms/blob/main/tbxforms/static/sass/components/_hint.scss#L5) - does this avoid this issue?

These being non-block elements (i.e. `<span>`) prevents forms from displaying as they're intended, as GOV.UK use `<p>`: https://design-system.service.gov.uk/components/text-input/#hint-text - so this is definitely a change we want to make, though happy to call this out in the upgrade considerations if helpful (after taking in to account the above)?

Reply:
Calling it out in the upgrade considerations sounds a good idea - happy with that.

Reply:
Great, I've added these to the WIP upgrade considerations which will be published alongside the next release. Danke!